pkg/proto/transactions.go

```diff
@@ -532,7 +539,7 @@ func NewUnsignedGenesis(recipient WavesAddress, amount, timestamp uint64) *Genes
 }
 
 // Validate checks the validity of transaction parameters and it's signature.
-func (tx *Genesis) Validate(scheme Scheme) (Transaction, error) {
+func (tx *Genesis) Validate(params TransactionValidationParams) (Transaction, error) {
 	if tx.Version < 1 || tx.Version > MaxGenesisTransactionVersion {
```

Comment:
I think max version of any transaction should be presented as a constant.

Reply:
Don't get you. There is a constant for max version of every transaction type.